Source/WebInspectorUI/UserInterface/Controllers/DOMManager.js

```diff
@@ -875,6 +875,10 @@ WI.DOMManager = class DOMManager extends WI.Object
             return;
 
         for (let target of WI.targets) {
```

Comment:
Should we switch this one as well?

Reply:
Seems like this is only done once in this file so I'll keep it for now.